src/views/Arbitrum.js

```diff
@@ -96,18 +89,24 @@ function Arbitrum(props) {
     if (!volumeData) {
       return null
     }
-    return volumeData[volumeData.length - 1].all
+    try {
+      return volumeData[volumeData.length - 1].all
```

Comment:
is try/catch needed here? in case volumeData is empty we can just check its length

Reply:
I don't think we need try catch. I have removed that.